README.md

```diff
@@ -213,6 +213,14 @@ See [`frontend/dockerfile/docs/experimental.md`](frontend/dockerfile/docs/experi
 
 By default, the build result and intermediate cache will only remain internally in BuildKit. An output needs to be specified to retrieve the result.
 
+> Color Output Controls
```

Comment:
This output section is for the exporters, e.g. exporting to docker images, or tarballs, etc, so doesn't feel like it quite fits here. Not quite sure where in the README is best, maybe right above it, or nearer the bottom around the opentracing support notes?